# Linear and angular velocity of the ball in polar coordinates
std_msgs/Header header
float64 module
float64 angle
